blog/tutorial/01-index.md

```diff
@@ -432,7 +432,7 @@ Now you are ready to build and start the app and create some posts.
 To launch the application run:
 
 ```
-starport serve
+starport chain serve
 ```
 
 This command installs dependencies, builds and initializes the app, and runs servers. You can also do it manually:
```

Comment:
```suggestion
This command installs dependencies, builds and initializes the app, and runs servers. You can also do these tasks manually:
```

Reply:
another unclear reference "it"
because we have a complex subject matter, it's very helpful to clarify the subject instead of using "it"